givitsite/gatherer/agora.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import urllib.request
+import urllib
+from bs4 import BeautifulSoup
+import re
+import csv
+from datetime import datetime
+from friendreq.models import ItemRequest
+
+
+def givit_main():
+    iseek_dict = {'20016': 'מיטה', '20008': 'כיסא', '20009': 'ארון',
+                  '20017': 'ספה', '10029': 'מכונת כביסה', '10006': 'מקרר'}
+    region_dict = {'Tel Aviv': 'תל אביב-יפו', 'Jerusalem': 'ירושלים',
+                   'Beer Sheva': 'באר שבע', 'Haifa': 'חיפה', 'Modiin': 'מודיעין', 'Hasharon': 'הוד השרון'}
+
+    items_list = get_items_list()
+    i = 0
+    for i in range(len(items_list)):
+        iseek = items_list[i][2]
+        my_soup = Agora_Getrequest(iseek)
+        url_list = find_furniture(
+            my_soup, region_dict.get(items_list[i][3]), iseek_dict.get(items_list[i][2]))
```

Comment:
This code could be much easier to understand if we had nice attribute names instead of things like `[i][2]`

Also you're using the dict's `get` method which means you might be passing `None` to the function when you don't find an item in the dict. Wouldn't in be better to let Python throw an exception when things are missing from the dict? It seems to me that that situation would need to be something that you as developers would need to know about and fix as early as possible... 

But then again, you should probably just let the model classes do this kinds of data format translations for you (E.g. from numbers to item kinds).